File: src/main/resources/output/IFC4X3_csharp/Entities/IfcResourceLevelRelationship/IfcMaterialRelationship/IfcMaterialRelationship.cs
public class IfcMaterialRelationship : IfcResourceLevelRelationship
{
    public IfcMaterial RelatingMaterial { get; set; }
    public SET [1:?] OF IfcMaterial RelatedMaterials { get; set; }
    public IfcLabel MaterialExpression { get; set; }
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcResourceLevelRelationship/IfcMaterialRelationship/IfcMaterialRelationship.py
class IfcMaterialRelationship(IfcResourceLevelRelationship):
    def __init__(self):
        self.RelatingMaterial: IfcMaterial = None
        self.RelatedMaterials: SET [1:?] OF IfcMaterial = None
        self.MaterialExpression: IfcLabel = None
